### DISCUSSION TITLE
copy and paste text from sumatra

### DISCUSSION THREAD

**Discussion by jacobkasper** [sumatrapdfreader/sumatrapdf#3840] [Q&A]
I am switching from Acrobat to Sumatra. When I copy text from a pdf in Sumatra the spaces are replaced with either "!" or if text is bold with "$". This does not happen when copying text from Acrobat in the same document. How do I resolve this issue?

**Comment by kjk** [comment]
Submit a bug report with a sample document as per https://www.sumatrapdfreader.org/docs/How-to-submit-bug-reports

**Comment by GitHubRulesOK** [comment]
issue with sample is #3844
